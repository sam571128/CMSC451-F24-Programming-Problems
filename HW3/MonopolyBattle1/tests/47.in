26 496 145
6 22
17 26
24 26
7 15
25 24
20 25
6 11
1 2
7 6
4 25
24 7
18 13
14 17
19 23
13 6
3 10
11 14
24 15
5 12
20 10
6 21
14 22
2 1
8 6
14 14
23 3
14 6
20 6
9 5
11 11
4 4
12 10
26 14
12 9
19 9
25 19
20 2
1 5
12 3
2 9
10 5
8 22
15 17
7 9
25 9
15 26
1 10
5 11
5 26
25 16
11 24
20 7
3 17
2 3
4 12
14 26
23 4
11 3
2 5
5 8
6 2
18 4
10 13
6 9
25 1
5 5
17 3
22 2
12 14
1 6
24 9
24 6
15 20
12 21
21 12
13 15
17 18
17 2
24 19
18 2
10 4
25 23
17 1
22 25
1 3
13 12
15 5
21 19
12 7
9 4
6 3
15 22
24 2
25 7
18 7
21 7
23 25
7 5
20 26
1 19
24 18
15 19
10 20
2 18
8 2
22 9
3 19
4 11
13 4
13 17
6 24
21 1
15 16
9 7
25 8
22 18
14 20
18 26
1 9
23 9
22 8
4 7
7 22
16 20
19 2
23 15
3 21
7 7
16 23
23 26
8 18
6 1
23 6
14 10
26 25
16 16
24 10
2 24
7 19
23 1
7 12
8 21
3 18
12 2
9 6
7 13
10 21
8 3
19 17
4 6
9 26
3 6
13 13
4 5
20 16
6 23
26 10
2 11
21 21
8 23
19 13
14 7
15 6
18 18
7 1
2 21
26 11
20 5
12 11
16 10
23 7
8 5
1 1
6 8
9 10
16 24
10 7
12 26
6 19
2 16
21 4
14 5
16 9
2 12
17 12
15 4
22 3
2 19
17 24
5 9
22 1
15 10
11 25
6 13
3 12
2 15
7 2
13 1
13 10
7 24
13 25
11 13
2 22
19 8
10 2
1 20
8 14
25 21
3 15
18 15
15 15
6 10
12 1
7 20
1 24
18 19
17 11
15 13
24 24
17 4
19 25
18 21
1 25
15 18
3 4
24 8
12 19
8 8
3 7
15 7
14 25
11 4
18 8
10 10
21 11
23 22
20 18
13 7
19 4
24 23
24 16
24 14
4 2
9 19
22 4
19 1
24 25
19 12
4 23
7 4
6 15
12 22
18 12
4 24
3 5
4 3
3 8
23 11
26 26
11 16
1 15
7 17
7 10
24 5
23 8
13 11
11 10
25 10
23 23
13 20
15 14
17 10
19 3
1 11
8 1
4 10
21 2
12 13
18 10
11 7
3 11
15 2
3 1
16 2
5 4
8 11
23 17
20 21
21 6
21 22
25 4
24 17
26 19
16 26
10 6
5 13
7 11
5 22
19 21
16 17
2 14
23 21
2 6
17 13
22 16
11 5
23 24
11 18
19 24
5 23
13 26
26 16
1 13
6 17
14 8
16 8
7 21
4 9
5 1
6 20
12 23
8 25
16 1
18 22
26 23
22 24
19 26
9 2
16 13
20 15
18 23
10 22
14 21
16 4
18 16
9 15
21 10
3 9
14 13
16 18
1 14
12 18
18 17
10 1
13 8
8 13
26 9
26 18
14 12
20 1
12 15
18 1
21 26
25 17
4 13
4 17
8 10
9 1
10 19
1 22
5 6
9 11
13 5
22 23
5 3
14 18
5 16
15 9
22 12
16 6
22 22
19 7
17 21
3 2
13 3
12 4
13 22
6 6
20 20
24 22
25 22
15 8
8 19
1 12
17 22
13 23
20 13
10 8
15 25
22 14
6 12
21 18
25 3
5 25
21 3
6 16
19 11
21 24
15 12
24 12
18 3
20 24
23 19
10 16
4 18
10 12
19 19
16 7
16 21
21 8
21 25
18 14
23 20
15 1
8 7
13 16
20 12
17 15
22 13
5 10
25 12
8 9
14 16
1 8
2 20
17 5
22 21
19 6
17 17
18 25
13 24
4 20
23 16
17 6
14 3
16 22
14 1
8 15
10 15
9 25
12 12
12 20
20 8
6 5
13 9
1 4
16 14
25 15
7 18
3 14
13 18
3 3
21 9
21 20
14 24
11 17
15 21
2 26
10 3
18 6
14 11
26 15
3 25
11 19
1 23
8 17
5 19
16 3
11 1
4 8
7 8
12 8
2 2
9 20
17 14
23 12
4 16
9 22
24 21
5 24
3 22
5 7
11 20
5 21
26 17
3 16
15 3
8 16
21 23
11 21
1 21
6 18
22 15
12 25
1 16
11 26
