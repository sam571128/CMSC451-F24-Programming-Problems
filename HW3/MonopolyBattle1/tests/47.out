61 23 39 73 38 48 51 60 38 38 48 38 63 48 51 73 48 77 73 73 48 48 73 51 73 48 